README.md

```diff
@@ -2,64 +2,38 @@
 
 Service for normalizing evidence
 
-## Developer instructions
+## Installation
 
-The following sections include instructions specifically for developers.
+Install from [PyPI](https://pypi.org/project/evidence-normalizer):
 
-### Installation
-
-#### Pipenv
-For a development install, we recommend using Pipenv. See the 
-[pipenv docs](https://pipenv-fork.readthedocs.io/en/latest/#install-pipenv-today) 
-for direction on installing pipenv in your compute environment.
- 
-Once installed, from the project root dir, just run:
-
-```commandline
-pipenv shell
-pipenv lock && pipenv sync
+```shell
+python3 -m pip install evidence-normalizer
 ```
 
-#### Pip
-
-If you wish to install developer dependencies for `evidence.dev`:
-```commandline
-pip install evidence-normalizer[dev]
-```
+To install ETL dependencies:
 
-If you do not need the extra dependencies:
-```commandline
-pip install evidence-normalizer
+```shell
+python3 -m pip install evidence-normalizer[etl]
 ```
 
-### Backend Services
-
-Evidence Normalization relies on [Variation Normalization](https://github.com/cancervariants/variation-normalization) for normalizing Cancer Hotspots data. You will need to setup backend services and set the appropriate environment variables. See the [README](https://github.com/cancervariants/variation-normalization#variation-normalization) for more information.
-
-### Init coding style tests
+## Development
 
-Code style is managed by [flake8](https://github.com/PyCQA/flake8) and checked prior to commit.
+Clone the repo and create a virtual environment:
 
-We use [pre-commit](https://pre-commit.com/#usage) to run conformance tests.
-
-This ensures:
-
-* Check code style
-* Check for added large files
-* Detect AWS Credentials
-* Detect Private Key
+```shell
+git clone <https://github.com/cancervariants/evidence-normalization>
```

Comment:
@korikuzma fwiw this gives me a syntax error in zsh

Reply:
Oh good catch. I forgot to remove the angle brackets after putting this in a code block